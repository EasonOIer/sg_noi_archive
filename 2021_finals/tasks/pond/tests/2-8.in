1964 975
998859 998273 997691 996777 995748 995690 994861 993275 990098 988527 987813 987238 986560 986010 985182 984436 983965 983840 982564 981020 980175 980000 979737 979416 978211 976432 975555 974904 973658 973191 972957 972856 971910 971246 969959 969190 967301 965497 965075 964437 963118 962494 962257 962036 957615 957475 954816 954376 953130 952760 952238 951963 951487 949621 948943 948064 946536 946454 944106 943795 942289 941188 940407 937981 935824 935815 934780 934039 933743 933743 932689 929600 929179 927842 925468 924621 923264 922602 922573 921826 920674 917705 916174 915253 914691 913639 912674 912106 911839 909989 906294 905807 904557 903517 902405 901069 900474 900199 899165 898338 897026 895724 893018 892948 892858 892844 892813 892252 891012 890517 889241 888750 886085 884494 884326 884258 883498 882429 882011 881351 881285 881216 879161 878387 874969 874466 872111 871493 868210 867651 865818 864548 864527 864121 863157 862950 862387 857394 857364 856818 856437 856328 854927 854414 852732 852140 850867 849535 848591 848537 845180 843921 843454 842868 839929 838922 837551 837076 836659 836598 836497 835562 835505 833971 833738 833420 832987 832428 832394 830712 830294 830282 828687 827585 827298 826412 826216 825667 823531 821832 821526 820486 819627 817898 817413 817189 816546 816544 814723 813561 811427 811262 810837 810666 808041 807472 805358 804303 804202 802061 800093 799250 799074 797730 794171 793846 793693 793516 792818 791602 789740 789632 789337 788559 788305 788178 786359 782892 782569 781860 781229 780946 780871 780082 778214 778023 774243 773207 773018 772901 771491 771447 770996 769469 767738 766718 764268 763832 763464 762108 760977 759044 758563 757326 756395 754878 754847 754174 753045 751091 750971 750553 750408 750180 748559 747999 747770 746362 745465 743965 743526 742652 741400 740421 739028 738730 737425 737207 736292 735792 733737 730188 729664 728274 727669 727151 726955 725412 724378 724081 721758 721021 720830 720343 719535 718786 717601 716292 715705 713347 712329 712216 709853 709706 709583 708142 707890 703320 703178 703017 701959 700815 700289 697426 697129 696482 695305 694097 693483 688597 684727 684504 684240 684065 682242 681178 680767 680691 679073 678918 678200 677915 677655 677058 676890 676639 676499 676344 675637 674239 672608 669100 668410 667470 665988 665767 664146 663662 662243 660998 658832 657581 656821 656311 654934 654513 654476 653377 652978 652129 647534 646134 641838 641318 640111 639690 639155 639091 638712 637978 637497 636798 636497 633648 633222 633046 632024 631886 631839 628877 628213 626818 626490 625402 624763 623438 621162 621028 620575 620271 619837 617590 616613 616514 616111 614902 614615 614361 613941 613498 612775 611499 611088 611051 610668 609867 609673 608932 608171 607239 607239 604164 603718 603383 602252 598576 596528 595881 595730 595528 594603 593548 591917 588765 588263 588099 584825 582090 577853 574829 574684 574430 573167 572893 572507 570861 570483 568374 567955 567618 567610 567183 566895 566538 565085 563274 563183 562283 561937 560263 558806 558432 558286 558229 555479 554720 554152 553842 553743 552604 548911 546899 545823 545156 545061 544327 543639 540975 540817 540390 538373 538087 536944 535754 535694 535504 534556 533794 533006 532707 532413 532138 531685 529562 528659 528248 527336 526682 525934 524800 524581 523732 523045 523021 522913 519445 517505 515265 513843 511548 511386 510590 508945 508402 507524 505953 505507 502713 501940 501759 500714 497871 495306 494761 494754 493211 492618 488457 488273 487835 485683 485480 485453 485386 484637 484397 484134 483587 483346 483153 481911 476978 474123 473793 473007 472063 469831 469475 469131 468889 464719 464654 464401 463472 463322 461360 457546 457187 456966 454135 454104 453952 452968 452555 452085 450330 447753 447275 446434 446021 445422 445168 444450 442671 441800 441506 441399 440813 440044 439999 439228 437868 436794 435872 435317 434147 432904 432777 431747 428321 427451 423890 423845 423298 423151 422676 420206 419997 418093 417121 416712 416373 414559 412916 410759 409522 409435 408849 408801 408516 408112 405737 404355 404319 401599 401367 400551 399861 398422 397113 395900 394285 392762 391765 391102 388991 388343 387807 387306 385891 385877 385760 384595 384589 383583 383084 382661 382336 381007 379080 378971 377964 374458 374133 373269 372716 372050 371537 370535 370141 369754 368877 365172 362680 362353 359252 357297 354115 352499 352157 347305 346723 346568 346365 345431 342071 339158 339084 337965 337924 337464 336376 336082 335381 334963 333294 333234 332824 328810 328791 327957 327424 327289 326158 325856 325512 321978 320372 318736 318663 317113 316761 316371 316340 315715 312391 312289 311506 309811 307525 307218 305289 304750 304343 303133 302911 301736 301404 299264 298214 297686 296016 295879 295425 295103 294992 294956 294653 294186 292979 292821 292243 290016 288763 288146 287995 286754 286384 286373 284766 282504 282499 281990 280220 279728 279588 276245 276196 275503 274293 273348 272693 270230 268839 268667 268520 266595 266003 265862 265554 265493 264081 263749 262073 261904 261431 260171 258524 256251 255138 254920 253359 250720 249870 248561 247696 247646 245599 244000 241074 240560 239614 238617 236937 235612 234873 234388 233849 232739 231964 231473 230975 228165 227999 227283 227224 227064 226910 226519 226314 224980 224496 221241 219631 219038 218598 217423 215084 214916 213512 213317 213312 211811 211635 210856 210632 210441 209926 209375 209172 209073 206826 206690 206072 205451 201666 200087 199944 199255 199090 197988 196689 195806 195735 193652 192828 192257 190480 187310 186704 185397 185302 184465 184100 182983 182429 178391 176218 176055 175076 173472 173001 172441 171256 170460 167318 167160 165549 164920 164637 164028 163553 163170 160780 160130 157772 157743 157155 155722 155695 154295 153977 153092 153049 152970 152359 150179 147502 146657 146546 145272 142028 141382 141308 140889 138729 137426 137331 136055 133338 131878 130800 128341 127998 126700 123096 122504 118373 117804 113164 110950 110412 109832 109503 109339 109139 107695 107317 107204 107176 106790 105952 104145 100467 100241 99498 99284 97223 94952 93659 92946 92788 92551 89611 88774 86492 84925 84119 82140 81575 81405 80993 76919 76912 76431 75882 74519 73927 73151 72277 71764 70571 69121 68801 67842 65249 62458 61927 60505 60245 57836 57758 55636 54506 53971 53764 53318 53031 52594 51840 48155 47880 45241 44433 43807 43002 42677 42442 39512 38435 36933 35880 35771 35750 34541 32781 31198 28461 27610 27369 27185 27001 26840 26541 24644 22119 21059 20239 19979 19827 19288 18690 17412 14882 14119 14041 13253 9779 9484 7397 6521 5173 4859 3155 40 445 1056 1601 1756 2250 2281 2784 3176 4574 4659 5846 6025 6588 6752 9092 13279 14072 14611 15350 15521 16700 20921 21381 22407 22898 22941 23128 23626 25834 30850 31870 32746 34530 34769 35688 36278 36359 37226 38012 38929 39108 39408 41595 41832 43218 43365 43926 44015 44618 46735 48112 49129 49826 50884 51761 52362 54954 56226 59201 60474 60599 61931 62337 62680 63181 64428 65021 65202 66828 67769 67911 68361 68472 69273 71082 71372 71968 74550 74682 75829 76140 77814 78127 78789 82457 83104 83148 84435 85438 86458 87411 87414 87598 88190 89389 91186 91295 91343 93072 93386 93432 93513 93519 93798 94482 97885 100878 101018 101755 101979 102787 104891 105157 106017 106174 106227 107511 109278 111595 111953 113219 114268 116267 116542 116844 117122 121472 122125 122526 122986 123153 123311 123907 126722 127674 128558 129214 129897 131447 133071 133269 133779 133858 134379 134891 135299 135797 136811 137636 138027 138763 138856 139291 139350 141684 142659 143018 143028 143273 144938 145859 145934 148727 150642 153015 154923 155507 155702 156184 158602 159900 161045 161402 161563 162118 164003 164230 164684 166237 166972 167598 168048 168271 172039 172718 173336 174939 175282 175774 175947 177890 178517 178718 178771 179257 179519 179970 179989 181192 181692 182338 182510 183068 185005 185272 186654 186749 187188 187285 188488 190172 190821 191478 192346 193827 194491 194609 194721 197723 198511 199584 200041 200268 200735 201333 201374 201841 203090 203100 203571 204426 206225 207586 209590 210682 212309 213217 215607 216154 216777 217083 217520 218373 220610 220985 222828 224453 226933 227401 227888 228826 229119 229283 229667 230653 232016 232490 233810 233910 234108 236517 237039 237216 239679 240960 242500 243599 244674 246576 246602 249163 249617 251618 251815 254233 254301 254310 254522 254546 256594 257992 259795 260348 260719 262604 262831 264252 265853 267313 267502 269735 272559 274213 277273 278058 278607 278773 279517 280025 281424 282596 282610 282809 282903 283242 285416 285667 286672 287060 287888 288700 289736 291824 291890 293279 293863 294138 295089 295585 297166 298165 298390 298979 299109 299290 299506 299833 301237 303999 304538 307446 309843 310434 310460 311618 313261 313558 313960 314144 314152 315271 315888 317188 317673 319472 320684 323325 323505 324268 324327 324716 325838 326226 327533 328792 331092 333465 334089 334437 334687 335622 336895 337117 337770 338400 339145 341247 341480 341492 343428 343541 344100 344817 347983 347984 348596 350574 350702 351130 352340 352884 353889 354085 355173 357732 357872 358833 359576 360005 362071 362886 364498 364782 365600 365608 365619 367835 368544 368726 370187 371450 372020 373235 373357 376540 376638 377363 377968 379582 379646 380289 381681 382569 384110 384400 387946 388238 388647 389013 389713 390008 390470 391407 391569 393780 395394 400307 401603 402378 406131 409178 412043 413349 414203 414512 414853 415443 416445 416952 418053 418779 419350 421377 422951 423815 424301 428148 430335 432568 433205 433515 434020 435143 435694 439023 439512 439876 449207 450578 451705 452107 452450 453164 453407 455563 456072 457517 457914 458502 459467 467077 471692 472527 472775 477749 478523 480101 481220 481615 482887 483462 484169 484243 485780 485916 486342 487756 488183 490613 491341 491626 493639 494073 494996 495508 497010 497272 501007 501936 502174 503134 504994 505814 507579 508825 510855 510898 511311 511506 511688 511932 512587 514410 515031 515328 515611 516470 517032 517660 519000 520596 521037 521118 521382 522811 524331 526613 527537 529163 529392 530149 530174 530493 530608 531250 532100 534359 537584 538230 538307 540787 542311 543459 544220 548071 548487 549720 550207 551493 554993 555039 556538 558082 560712 561221 561573 562353 562653 563298 563946 565843 567031 569528 571850 572098 573642 573836 574021 576279 577227 577296 577319 578497 578708 579359 579745 579989 582255 583571 583949 584726 584766 589438 589486 590259 591457 591546 591670 592836 595159 595553 595968 597232 597326 597483 600860 601037 601116 601190 602795 603101 605378 607420 607448 607773 607894 610826 612393 612596 612788 613583 613933 615933 616001 616771 616847 617794 618537 618597 619272 620035 620296 621222 621488 621674 622029 622330 622720 623568 624637 625227 625270 625737 626084 626229 627197 627664 628153 628828 632699 633158 633203 636738 637709 637856 638047 638380 638584 638716 642372 645987 646929 647413 650024 652348 652877 655970 657037 657393 660456 660699 661245 662051 662291 663193 664306 668093 668426 669196 671106 671233 672519 672910 674806 675065 676642 676700 677631 678839 679855 679875 680055 683869 685302 685374 686513 689476 691308 693340 694126 694221 695965 696480 696603 698010 698221 698484 698704 702203 703106 704790 704986 708937 715905 716563 718147 719243 719740 721499 722665 722805 723337 723695 724367 725178 727143 728641 731286 731687 732606 734838 735613 735770 736892 736937 737615 737777 737813 738053 739613 739758 739829 742900 744570 745184 745800 746503 747572 747735 748129 748215 748244 749584 751854 752125 752196 752353 754209 754715 755078 756107 756482 757706 758320 758652 759252 761853 763496 764747 766086 767352 768338 768471 768811 769221 770086 771358 771740 772426 773074 773774 775290 776951 777312 778387 779976 780890 784228 785814 788339 788877 789299 791224 792011 793032 795053 795460 796561 799673 802355 803248 803990 804263 806367 807171 807410 809001 809480 810075 810576 810867 814609 815895 816039 819300 820675 822451 822553 822623 823765 824972 831511 831602 832056 833040 834331 834394 835585 838616 838683 838955 839414 839439 839694 839874 840922 841181 841723 842284 842314 842397 843434 843525 843720 845494 845977 846132 848046 848432 849191 850379 851366 851768 851913 852508 852929 854028 854453 854584 854812 858258 858428 859563 860118 863424 864812 865357 865834 866454 867110 867570 868091 870615 871143 872311 872536 872674 873458 874479 874959 875047 875688 876117 876149 876543 879154 880024 881522 882103 883423 885390 885853 886787 887190 887411 887523 887691 889019 889208 893066 893969 894743 895324 895519 897305 897826 898454 899271 903422 903793 904085 904765 905960 906491 906947 906995 908771 909218 909326 910577 911905 912520 914101 917771 920754 921159 921582 922285 922643 926296 928008 929035 934640 936092 937298 938331 938614 939776 940404 940755 940999 943034 943244 943751 943996 944627 945639 946970 948243 949171 949863 950281 952964 953327 953586 955159 956318 956626 956633 956720 958262 959763 959894 961603 964219 964228 965481 967077 968300 973090 973916 974171 974760 974861 975001 976612 976753 977111 978863 980794 983094 985006 985182 985777 989061 989326 989341 990653 991040 991486 992477 992602 994685 996065 997047 997095 998592 999724
