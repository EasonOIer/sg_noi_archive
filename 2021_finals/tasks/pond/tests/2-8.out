1132051317884